.travis.yml

```diff
@@ -20,53 +20,75 @@ services: docker
 
 matrix:
   include:
-    - os: linux
-      env: MB_PYTHON_VERSION=2.7
```

Comment:
any reasons why you drop python2.7? It should not be related to your specific change

Reply:
It is no more maintained by python community, so I removed it. Please let me know if it's required, I will reverse the changes.